docs/src/cfe_api.dox

```diff
@@ -105,16 +124,83 @@
     </UL>
     <LI> \ref CFEAPIFSUtil
     <UL>
+          <LI> #CFE_FS_GetDefaultMountPoint - \copybrief CFE_FS_GetDefaultMountPoint
+          <LI> #CFE_FS_GetDefaultExtension - \copybrief CFE_FS_GetDefaultExtension
+          <LI> #CFE_FS_ParseInputFileNameEx - \copybrief CFE_FS_ParseInputFileNameEx
+          <LI> #CFE_FS_ParseInputFileName - \copybrief CFE_FS_ParseInputFileName
 	  <LI> #CFE_FS_ExtractFilenameFromPath - \copybrief CFE_FS_ExtractFilenameFromPath
+          <LI> #CFE_FS_BackgroundFileDumpRequest - \copybrief CFE_FS_BackgroundFileDumpRequest
+          <LI> #CFE_FS_BackgroundFileDumpIsPending - \copybrief CFE_FS_BackgroundFileDumpIsPending
+    </UL>
+  </UL>
+
+  <H4> Message API </H4>
+  <UL>
+    <LI> \ref CFEAPIMSGHeader
```

Comment:
Would it be worth making these ref names match the naming convention using underscores? Or is this a doxygen quirk?

Reply:
Group names can be whatever you want as far as I know, feel free to update.